pkg/autodiscovery/autoconfig.go

```diff
@@ -454,6 +411,55 @@ func (ac *AutoConfig) pollConfigs() {
 	}()
 }
 
+// TODO: move to check scheduler
+func (ac *AutoConfig) processRemovedConfigs(removedConfigs []integration.Config) {
+	// Process removed configs first to handle the case where a
```

Comment:
I would move this comment to the where this method gets called in `pollConfigs `.